rust/macros/module.rs

```diff
@@ -724,6 +732,149 @@ pub fn module_misc_device(ts: TokenStream) -> TokenStream {
     .expect("Error parsing formatted string into token stream.")
 }
 
+pub fn rtnl_link_ops(ts: TokenStream) -> TokenStream {
+    let mut it = ts.into_iter();
+    let literals = vec!["maxtype", "policy", "slave_maxtype", "slave_policy"];
```

Comment:
There is no need to allocate a vector here. A slice or maybe an array would work fine too.

Reply:
It's only built time, so should not harm. But I will change it.